documentation/how_to_format_cpp_lambda_functions.rst

```diff
@@ -2,75 +2,59 @@
 Formatting C++ Lambda Expressions
 #################################
 
-Uncrustify supports some formatting of c++ lambda expressions, although the support is incomplete.
-A complete description of the c++ lambda expressions 
-The parts of c++ lambda expressions that is currently recognized by Uncrusitify is:
+Uncrustify supports some formatting of c++ lambda expressions, although 
+the support is incomplete.
+The parts of c++ lambda expressions that are currently recognized by Uncrusitify
+ are:
 
 .. code-block:: c++
 
   [ captures ] execution_context ( params ) specifiers -> ret { body }
   
-Explanations for all the tokens in the above lambda expression (except for execution_context) are found `here <https://en.cppreference.com/w/cpp/language/lambda>`_.
-The `execution_context` token is a non-standard addition to allow for specification of the execution space (e.g. host or device in CUDA).
-The native specifiers for the `execution_context` for lambda expression in CUDA are ``__device__`` and ``__host__ __device__``.
-However it is common for codes to use a preprocessor variable in place of the native specifiers.
+Explanations for all the tokens in the above lambda expression (except for 
+execution_context),
+as well as a complete description of c++ lambda expressions are found
+`here <https://en.cppreference.com/w/cpp/language/lambda>`_.
+The `execution_context` token is a non-standard addition to allow for 
+specification of the 
+execution space (e.g. host or device in CUDA).
+The native specifiers for the `execution_context` for lambda expression in CUDA 
+are 
+``__device__`` and ``__host__ __device__``.
+However, it is common for code to use a preprocessor variable in place of the 
+native specifiers.
+
+The Uncrustify options for formatting of c++ lambda expressions are:
+
+.. code-block::
+
+  sp_cpp_lambda_assign
+  sp_cpp_lambda_square_paren
+  sp_cpp_lambda_square_brace
+  sp_cpp_lambda_paren_brace
+  sp_cpp_lambda_fparen
+  nl_cpp_lambda_leave_one_liners
+  nl_cpp_ldef_brace
+  indent_cpp_lambda_body
+  indent_cpp_lambda_only_once
+
+Please refer to the example configuration file at 
+`uncrustify/documentation/htdocs/default.cfg <https://github.com/uncrustify/uncrustify/blob/master/documentation/htdocs/default.cfg>`_ 
```

Comment:
Ick. Oh, well, there really isn't anything that can be done with this :slightly_smiling_face:.